CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Ember CLI Mirage Changelog
 
+## next
+
+Changes:
```

Comment:
We may want to add a deprecation warning for this in v0.0.30?
And then actually do the release in v0.1.x